edb/lib/schema.edgeql

```diff
@@ -275,7 +276,8 @@ CREATE ABSTRACT TYPE schema::Source EXTENDING schema::Object {
 
 CREATE ABSTRACT TYPE schema::Pointer
     EXTENDING
-        schema::InheritingObject, schema::ConsistencySubject,
+        schema::ConsistencySubject,
+        schema::InheritingObject,
```

Comment:
You made `ConsistencySubject` an `InheritingObject`, so perhaps drop the entry here?